### PR TITLE
Add a mobile runs section

### LINKED ISSUE #82: Add a mobile runs section
We have stable and experimental runs now, but we should add a mobile runs section as well to notice breakage in Android runs today (Chrome & Firefox) and iOS (Safari) in the future.

### PR DESCRIPTION
**Pull request by past**
Fixes #82

### CODE REVIEW

**Comment by past** [comment]
It includes a small cleanup from https://github.com/Ecosystem-Infra/ecosystem-infra-rotation/pull/62 leftovers.

File: docs/wpt.fyi-mobile.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Runs for mobile Chrome and Firefox should happen at least daily.
+
+If this check fails, file an issue on [wpt.fyi](https://github.com/web-platform-tests/wpt.fyi) if all runs are affected, or [wpt](https://github.com/web-platform-tests/wpt) with the [infra label](https://github.com/web-platform-tests/wpt/labels/infra) if only some runs are missing. To check for issues on recent runs go to [wpt.fyi/status](https://wpt.fyi/status), select the `Invalid runs` tab, and then click on `Show error`. If the issue is with the WPT CI, then it's best to file an issue in the wpt repo, even in the case of all engines being affected.
+
+Contact points for the underlying runner infrastructure:
+* Chrome - [@WeizhongX](https://github.com/WeizhongX) (Chromium CI)
+* Firefox - [@jgraham](http://github.com/jgraham) (Taskcluster)
+* Safari - [@gsnedders](https://github.com/gsnedders) (Safari CI)
```
Comment (jcscottiii) [inline docs/wpt.fyi-mobile.md:8]: Thoughts on removing Safari CI for now since the query below only includes firefox and chrome?

**Review by jcscottiii — APPROVED** [review]
The query is valid and returns the list of runs for chrome android and firefox android.
I just had one small suggestion / question.

LGTM.

File: docs/wpt.fyi-mobile.md (hunk 1 above)
Comment (past) [reply docs/wpt.fyi-mobile.md:8]: Yeah, fair enough.

**Pull request merged**